connectors/data.json

```diff
@@ -27,87 +26,28 @@
         },
         "configurations": [],
         "dataImports": [],
-        "rpm_name": "cyops-connector-carbonblack-response-2.0.1",
-        "install_mode": "rpm",
-        "installer_path": "cyops-connector-carbonblack-response-2.0.1"
+        "install_mode": "rpm"
     },
     {
         "name": "ipstack",
         "label": "IPStack",
         "category": "Information",
         "description": "IPStack provides geolocation facility for IP Address or Domain.",
         "publisher": null,
```

Comment:
Why we are adding connector publisher here?

Reply:
and category?

Reply:
and description?

Reply:
IMO, its not needed. If anything changes in connector will impact this json as well. Please check with Aakash and Pooja.